src/jinja2/environment.py

```diff
@@ -855,7 +949,62 @@ def _load_template(self, name, globals):
         return template
 
     @internalcode
-    def get_template(self, name, parent=None, globals=None):
+    def _check_multi_template_autoescape(
+        self,
+        name: t.Union[None, str, "Template"],
+        parent: t.Optional[str],
+        caller: t.Optional[str],
+    ) -> None:
+        """
+        Raise an Error if we expect different custom escape functions
+        or Markup classes used in a way that a user could get
+        unexpected results.
+
+        :param name: The name of first template if known
+        :param parent: the name or names of the templates that
+        :param caller: the name of the function the
+
+        :raises TemplateConfigurationError
```

Comment:
```suggestion
        :raise: TemplateConfigurationError
```

(or `:raises:`, see what's more common in the codebase)